cmake/DaemonFlags.cmake

```diff
@@ -162,7 +162,7 @@ elseif (NACL)
     set_c_cxx_flag("-O2 -DNDEBUG -g3"   RelWithDebInfo)
     set_c_cxx_flag("-O0          -g3"   Debug)
 else()
-    set_c_cxx_flag("-ffast-math")
+#    set_c_cxx_flag("-ffast-math")
```

Comment:
what's this

Reply:
You noticed it between two force-push, faster than I notice it myself :sweat_smile:, was an attempt for https://github.com/DaemonEngine/Daemon/issues/169#issuecomment-611310651, was mistakenly committed there.